# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/fake/mythphone/use-system-avcodec.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./mythphone/h263.h.orig	2005-01-08 02:48:52.326346616 +0100
+++ ./mythphone/h263.h	2005-01-08 02:48:59.197302072 +0100
@@ -26,7 +26,7 @@
 #ifdef WIN32
 #include "libavcodec/avcodec.h"
 #else
-#include "mythtv/ffmpeg/avcodec.h"
+#include <ffmpeg/avcodec.h>
 #endif
 }
 
